### PR TITLE
dev/sg: log gh cli version

### PR DESCRIPTION
**Pull request by michaellzc**
## Test plan

<!-- All pull requests REQUIRE a test plan: https://docs.sourcegraph.com/dev/background-information/testing_principles -->

```sh
go run ./dev/sg cloud install         
                                                                                                                                                                                                                                                                        
👉 Removing existing installation at of "mi2" at "/Users/michael/.bin/mi2"
👉 Using GitHub CLI version "gh version 2.23.0 (2023-02-07)"
```

### CODE REVIEW

**Comment by michaellzc** [comment]
Current dependencies on/for this PR:
* main
  * **PR #48853** <a href="https://app.graphite.dev/github/pr/sourcegraph/sourcegraph/48853" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sourcegraph/sourcegraph/48853?utm_source=stack-comment).

File: dev/sg/sg_cloud.go [hunk 1]
```diff
@@ -60,6 +60,12 @@ var cloudCommand = &cli.Command{
 					return errors.Wrap(err, "GitHub CLI (https://cli.github.com/) is required for installation")
 				}
 
+				version, err := run.Cmd(c.Context, "gh", "version").Run().Lines()
```
Comment (bobheadxi) [inline dev/sg/sg_cloud.go:63]:
```suggestion
				version, err := run.Cmd(c.Context, "gh", "version").Run().String()
```

**Review by bobheadxi — APPROVED** [review]

File: dev/sg/sg_cloud.go (hunk 1 above)
Comment (bobheadxi) [reply dev/sg/sg_cloud.go:63]: then you don't risk e.g. edge cases with no output panicking

**Review by bobheadxi — APPROVED** [review]

File: dev/sg/sg_cloud.go (hunk 1 above)
Comment (michaellzc) [reply dev/sg/sg_cloud.go]: updated, I ended up splting the string manually to avoid panic from `run`

**Pull request merged**